libmscore/segment.cpp

```diff
@@ -1165,13 +1165,29 @@ void Segment::scanElements(void* data, void (*func)(void*, Element*), bool all)
       {
       for (int track = 0; track < score()->nstaves() * VOICES; ++track) {
             int staffIdx = track/VOICES;
-            if (!all && !(measure()->visible(staffIdx) && score()->staff(staffIdx)->show())) {
+            if (!all && !(/*measure()->visible(staffIdx) &&*/ score()->staff(staffIdx)->show())) {
```

Comment:
Should the commented out code be removed or reinstated?

Reply:
That would be a check if the measure is marked as visible. I'll double check. Thanks for helping me by reviewing the code.

Reply:
Removed it.